docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -89,6 +89,26 @@ Python style
         """
         ...
 
+* In many places in Django's codebase, we catch low-level exceptions that have
+  been raised, and raise our own exception instead-- Also known as "wrapping
+  exceptions." The new exception will often have a different type and message.
+  That makes it easier to catch, understand and fix. Example::
```

Comment:
`makes it easier for users to catch, understand and fix. For example::`

Reply:
Fixed.